src/web/resources/report.js

```diff
@@ -35,91 +35,135 @@ function Element(tagname, props, children) {
     return $elt;
 }
 
-var renames = {};
-renames["imp-start"] = "Improved start"
-renames["apx-start"] = "Approximate start"
-renames["uni-start"] = "Regressed from start"
-renames["ex-start"]  = "Exact start"
-renames["eq-start"]  = "Equal start"
-renames["lt-start"]  = "Less than start"
-renames["gt-start"]  = "Greater than start"
-renames["gt-target"] = "Greater than target"
-renames["eq-target"] = "Equal than target"
-renames["lt-target"] = "Less than target"
-renames["error"]     = "Error"
-renames["timeout"]   = "Timeout"
-renames["crash"]     = "Crash"
-
-const Filters = new Component("#filters", {
-    setup: function () {
-        // Warning fragile starting state
-        const regressedTags = ["uni-start", "lt-target", "lt-start", 
-        "apx-start", "timeout", "crash", "error"]
-        const improvedTags = ["imp-start", "ex-start", "eq-start", "eq-target",
-        "gt-target"]
-        const advancedTags = []
-        advancedTags.push(regressedTags)
-        advancedTags.push(improvedTags)
+const renames = {
+    "imp-start": "Improved start",
+    "apx-start": "Approximate start",
+    "uni-start": "Regressed from start",
+    "ex-start": "Exact start",
+    "eq-start": "Equal start",
+    "lt-start": "Less than start",
+    "gt-start": "Greater than start",
+    "gt-target": "Greater than target",
+    "eq-target": "Equal than target",
+    "lt-target": "Less than target",
+    "error": "Error",
+    "timeout": "Timeout",
+    "crash": "Crash",
+}
 
-        // build labels
-        const regressedLabel = this.buildCheckboxLabel("regressed", "Regressed", true)
-        const improvedLabel = this.buildCheckboxLabel("improved", "Improved", true)
+var resultsTestData = null
```

Comment:
I know globals are bad but I wasn't sure how else to "cache" the data from `results.json`